plotting/class4.py

```diff
@@ -14,6 +14,7 @@
 class Class4Plotter(Plotter):
     def __init__(self, dataset_name: str, query: str, **kwargs):
         self.plottype: str = "class4"
+        self.class4Type: str = None
```

Comment:
this is pretty nitpicky but stylistically python variables should use snake case. ie  
```suggestion
        self.class4_type: str = None
```